### PR TITLE
Changes made for progress bar to be responsive.

### PR DESCRIPTION
**Pull request by Amutha37**
Please view the progress bar changes made for responsive screen. I need discuss about the text on checkbox in today's meeting.

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -14,3 +15,72 @@ a {
...
+    font-size: 2px;
+  }
+  /* td > p {
+    font-size: calc(8px + 3vmin);
+    font-size: 2px;
+  }
+  .responsivebar {
+    height: "50";
+    width: "100%";
+  } */
```
Comment (hqtan) [inline styles/globals.css:47]: please remove commented out code
```suggestion
```

**Review by hqtan — CHANGES_REQUESTED** [review]
Hi @Amutha37 , Thx for the PR :bow:

I really like what you've done to the progress bar :heart_eyes:. The length of the progress bar seems more aligned with other components on each page.

I've pointed out a few other things in the code to bring to your attention.

Just for future reference, it would have been nice to see all the changes from y
our [earlier PR#24](https://github.com/CodeforAustralia/council-emissions-calculator-spike/pull/24) sorted out and merged in, rather than having the changes from PR#24 now brought into this PR; it's not as easy for the reviewer to tease out what recommendations from PR#24 that you have acted/not acted upon... but it's done, so let's move on this time round :smile:

**Comment by hqtan** [comment]
@batteries76 , @ymloh ,

what do you think about the changes related to making the progress bar responsive? 
I think I prefer this to #26  ... (but I'm not a frontend dev, so my opinion here may not matter as much :sweat_smile: )

for comparison: 
- here's a screenshot of progress bar for this PR:
image

- here's a screenshot of progress bar for #26
image

File: pages/form/Question1.js [hunk 2]
```diff
@@ -74,14 +113,26 @@ export default function Question1() {
...
-            <path fillRule="evenodd" clipRule="evenodd" d="M1.1875 7C1.1875 6.75136 1.28627 6.5129 1.46209 6.33709C1.6379 6.16127 1.87636 6.0625 2.125 6.0625H21.8125C22.0611 6.0625 22.2996 6.16127 22.4754 6.33709C22.6512 6.5129 22.75 6.75136 22.75 7C22.75 7.24864 22.6512 7.4871 22.4754 7.66291C22.2996 7.83873 22.0611 7.9375 21.8125 7.9375H2.125C1.87636 7.9375 1.6379 7.83873 1.46209 7.66291C1.28627 7.4871 1.1875 7.24864 1.1875 7Z" fill="var(--chakra-colors-blue-600)" />
+        <LinkButton href="/" onClick={saveAnswers} variant="outline">
```
[the remark this answers is not in the document]
Comment (Amutha37) [reply pages/form/Question1.js:106]: Will create a separate PR for <BackButton>

File: pages/form/Question1.js [hunk 3]
```diff
@@ -39,33 +49,62 @@ export default function Question1() {
...
-    }))
-  }
+    }));
+  };
 
   return (
     <Layout>
-      <Q1Progress /> 
+      <Q1Progress
+        className="responsivebar"
```
[the remark this answers is not in the document]
Comment (Amutha37) [reply pages/form/Question1.js:56]: Removed line 64 and 54.

File: pages/form/Question1.js [hunk 4]
```diff
@@ -39,33 +49,62 @@ export default function Question1() {
...
+                <Td fontWeight="bold" color="#333">
+                  {day}
+                </Td>
                 <Td>
-                  <Grid templateColumns="repeat(2, 1fr)" gap={6} color="#212529">
-                    <WorkCheckBox value="home" index={i}>Work from home</WorkCheckBox>
-                    <WorkCheckBox value="office" index={i}>From workplace</WorkCheckBox>
+                  <Grid
+                    // fontSize={["sm", "md", "lg", "xl"]}
+
```
[the remark this answers is not in the document]
Comment (Amutha37) [reply pages/form/Question1.js]: Removed line 94 and 95.

File: pages/form/Question1.js [hunk 5]
```diff
@@ -14,9 +13,20 @@ import {
   Th,
   Td,
   TableCaption,
+  createBreakpoints,
```
[the remark this answers is not in the document]
Comment (Amutha37) [reply pages/form/Question1.js]: Removed line 16.

File: pages/form/Question1.js [hunk 6]
```diff
@@ -14,9 +13,20 @@ import {
...
+import LinkButton from "../../components/LinkButton/LinkButton";
+import Q1Progress from "../../public/images/progress-bar/q1-progress-bar.svg";
+import { daysOfWeek } from "../../utils/constants";
+// const breakpoints = createBreakpoints({
+//   sm: "30em",
+//   md: "48em",
+//   lg: "62em",
+//   xl: "80em",
+//   "2xl": "96em",
+// });
```
[the remark this answers is not in the document]
Comment (Amutha37) [reply pages/form/Question1.js]: Removed all the commented lines.

File: styles/globals.css (hunk 1 above)
Comment (Amutha37) [reply styles/globals.css:47]: Removed all the commented line.

File: pages/form/Question1.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Amutha37) [reply pages/form/Question1.js:106]: Can the person who created teh LinkButton component look at this request.

**Comment by hqtan** [comment]
Hi @Amutha37 

Are you able to remove the changes for the download button in the results page, and move that to a separate PR please?

**Comment by Amutha37** [comment]
Hi Qing,

I will do that tomorrow. Sorry, I just saw your message.

Amutha

**Pull request closed without merging**